src/@commonsku/styles/DateRangePicker.tsx

```diff
@@ -119,155 +137,179 @@ export const DateRangePicker = forwardRef<HTMLDivElement, DateRangePickerProps>(
-                        selectsEnd
-                        startDate={startDate ?? (endDate ? new Date(0) : null)}
-                        endDate={endDate ?? (startDate ? new Date(9999, 1, 1) : null)}
-                        todayButton={todayButton}
-                        dateFormat={dateFormat}
-                        isClearable={isClearable}
-                        showMonthDropdown={showMonthDropdown}
-                        showYearDropdown={showYearDropdown}
-                        nextMonthButtonLabel={nextMonthButtonLabel}
-                        nextYearButtonLabel={nextYearButtonLabel}
-                        previousMonthButtonLabel={previousMonthButtonLabel}
-                        previousYearButtonLabel={previousYearButtonLabel}
-                        dropdownMode={dropdownMode}
-                        peekNextMonth={peekNextMonth}
-                        calendarClassName={`commonsku-styles-datepicker daterangepicker ${popperClassName || ''}`}
-                        onChange={(newEnd, event) => handleChange('end', startDate, newEnd, event)}
-                        {...props}
-                    />
-                </div>
-            </Col>
-        </Row>
+      <Row style={{ flexFlow: "row", gap: "2rem" }}>
+        <Col>
+          <div style={dateRangeLabelStyles}>From</div>
+          <div
+            style={{ display: "flex", flexDirection: "column", gap: "1rem" }}
+          >
+            <Datepicker
+              open={false}
+              locale={locale}
+              selected={startDate}
+              dateFormat={dateFormat}
+              isClearable={isClearable}
+              onChange={(newStart, event) => {
+                if (!checkDateYear(newStart)) return;
+                handleChange("start", newStart, endDate, event);
+              }}
+            />
+            <Datepicker
+              key={startDateKey}
+              inline
+              locale={locale}
+              selected={startDate}
+              selectsStart
+              startDate={startDate ?? (endDate ? new Date(0) : null)}
+              endDate={endDate ?? (startDate ? new Date(9999, 1, 1) : null)}
+              todayButton={todayButton}
+              dateFormat={dateFormat}
+              isClearable={isClearable}
+              showMonthDropdown={showMonthDropdown}
+              showYearDropdown={showYearDropdown}
+              nextMonthButtonLabel={nextMonthButtonLabel}
+              nextYearButtonLabel={nextYearButtonLabel}
+              previousMonthButtonLabel={previousMonthButtonLabel}
+              previousYearButtonLabel={previousYearButtonLabel}
+              dropdownMode={dropdownMode}
+              peekNextMonth={peekNextMonth}
+              calendarClassName={`commonsku-styles-datepicker daterangepicker ${
+                popperClassName || ""
+              }`}
+              onChange={(newStart, event) =>
+                handleChange("start", newStart, endDate, event)
```

Comment:
Is there a reason that you are not doing `if (!checkDateYear(newStart)) return;` here? (And in the equivalent place for the end date)